packages/dds/cell/src/cell.ts

```diff
@@ -295,4 +309,42 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
         ++this.messageId;
         return this.messageId;
     }
+
+    /**
+     * Rollback a local op
+     * @param op - The operation to rollback
+     * @param localOpMetadata - The local metadata associated with the op.
+     */
+    public rollback(op: any, localOpMetadata: ICellLocalOpMetadata) {
```

Comment:
This should be protected. You want to override the implementation in sharedObjectCore which is only called by sharedObjectCore.